src/sig/mayo/sig_mayo_1.c

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT
+
+#include <stdlib.h>
+
+#include <oqs/sig_mayo.h>
+
+#if defined(OQS_ENABLE_SIG_mayo_1)
+
+OQS_SIG *OQS_SIG_mayo_1_new(void) {
+
+	OQS_SIG *sig = malloc(sizeof(OQS_SIG));
+	if (sig == NULL) {
+		return NULL;
+	}
+	sig->method_name = OQS_SIG_alg_mayo_1;
+	sig->alg_version = "https://github.com/PQCMayo/MAYO-C/tree/nibbling-mayo";
```

Comment:
Yeah, I'm not sure what's the best descriptor there and other algorithms aren't always consistent in what they document there.

The `alg_version` field is documented as:
```Implementations with the same method_name and same alg_version will be interoperable.```
My rationale was that a new commit commit could be a minor fix still interoperable while the branch in MAYO defines the interoperable algorithm version. Not completely sure about this though, would the spec. version be more meaningful perhaps?

Reply:
Yeah... the PQClean implementations seem to use the spec version; Kyber, Dilithium, and SPHINCS+ all seem to use commit hashes; while ML-KEM and ML-DSA use branch names. My rationale was that branches are a "softer" tag than a hash: one can update the implementation on a branch such that it becomes non-interoperable with older implementations on the same branch, but this can't be done with a hash. As you point out, however, most commits don't break interoperability. If the MAYO team is committed to implementations on the `nibbling-mayo` branch being interoperable, then I agree that a branch name is preferable.

(As an aside, maybe we should add interoperability testing...)

Reply:
It's intended to not make breaking changes in the `nibbling-mayo` branch.